src/hallway/feed.clj

```diff
@@ -0,0 +1,29 @@
+(ns hallway.feed
+  (:require
+    [biff.util :as bu]
+    [hallway.util :as u]))
+
+(def xml-doc
+  "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n")
+
+(defn entry [{:keys [title url created author source]}]
+  [:entry
+   [:title title]
+   [:link {:href url}]
+   [:id url]
+   [:updated (u/format-date created)]
+   [:author [:name author]]
+   [:category
+    {:term source}]
+   [:summary title]])
+
+(defn feed [{:keys [params/url biff/base-url]} docs]
+  (let [sorted-docs (sort-by :created #(compare %2 %1) docs)
+        self (str base-url "/feed?url=" url)]
```

Comment:
(I should've renamed this file in a separate commit so you could see the individual changes I actually made...)

Originally this was "https://discuss.findka.com/...", but using `base-url` (a config value added by Biff) instead will make it easier for people to host on different domains if they want to.